crates/collab/src/db/queries/channels.rs

```diff
@@ -59,16 +59,28 @@ impl Database {
                 parent = Some(parent_channel);
             }
 
+            let parent_path = parent
+                .as_ref()
+                .map_or(String::new(), |parent| parent.path());
+
+            // Find the maximum channel_order among siblings to set the new channel at the end
+            let max_order = max_order(&parent_path, &tx).await?;
```

Comment:
I wonder if we should make the max order optional, based on whether the channel is a root channel or not. As written, this code will give every root channel an order that is unescessary.